library/freeb/xmlrpc_server.php

```diff
@@ -2,9 +2,7 @@
 require_once("xmlrpc.inc");
```

Comment:
anything using freeb should be removed entirely.   There might be one place in billing the saves files to that dir the need refactorung, but freeb XMLRPC has not be used since version 2.8